thundra/thundra_agent.py

```diff
@@ -59,7 +69,7 @@ def __call__(self, original_func):
 
         @wraps(original_func)
         def wrapper(event, context):
-            if self.checkAndHandleWarmupRequest(event):
+            if self.warmup_aware and self.checkAndHandleWarmupRequest(event):
                 constants.REQUEST_COUNT += 1
                 return None
```

Comment:
This code is not changed in this task, but I don't feel safe about returning None from a wrapper method. We can check it later.